Remove malloc.h include since it is nonstandard and does not exist on macOS.
https://sourceforge.net/p/xine/mailman/message/37402874/
https://sourceforge.net/p/xine/xine-ui/ci/3ea5f5a563782e4f2f5230e9d8e3a658229536a4/
--- src/xitk/xine-toolkit/backend_x11.c.orig	2021-12-12 13:49:29.000000000 -0600
+++ src/xitk/xine-toolkit/backend_x11.c	2021-12-16 23:59:47.000000000 -0600
@@ -30,7 +30,6 @@
 #include <strings.h>
 #include <unistd.h>
 #include <pthread.h>
-#include <malloc.h>
 
 #include <X11/Xlib.h>
 #include <X11/Xutil.h>
